modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/LoadCoreMembers.pm

```diff
@@ -145,6 +119,28 @@ sub pipeline_analyses_copy_ncbi_and_core_genome_db {
                 mode   => 'members_globally',
             },
         },
+    #--------------------Reference genome loading------------------#
+        {   -logic_name => 'copy_ref_genomes_factory',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::GenomeDBFactory',
+            -parameters => {
+                'compara_db'    => '#rr_ref_db#',
+                'all_current'   => 1,
```

Comment:
Something to consider later on rather than now is that a RR version will be used and the `software_version()` will be redundant, this may or may not have an impact on the `compara_references` - but non-current are unlikely to be kept in the db - more release style rather then `master`. Another future option could be to take the `genome_db` names from `references/allowed_species.json` file and copy these across instead.

Reply:
I think the best approach is to keep a clean reference database so there is no need to add such logic to this pipeline, but these are good thoughts to keep into account indeed.